charts/cloudzero-prometheus-agent/templates/_helpers.tpl

```diff
@@ -14,13 +14,13 @@ Create chart name and version as used by the chart label.
 
 {{/* Define the url to which metrics should be sent */}}
 {{- define "cloudzero-agent.remoteWriteUrl" -}}
-https://{{ .Values.global.cloudzeroHost }}/v1/container-metrics?cluster_name={{.Values.global.cluster_name}}&cloud_account_id={{.Values.global.cloud_account_id}}
+https://{{ .Values.global.cloudzeroHost }}/v1/container-metrics?clusterName={{.Values.global.clusterName}}&cloudAccountId={{.Values.global.cloudAccountId}}
```

Comment:
sorry, I think I added this named template but didn't use it. we should either get rid of it or use it in the configmap

Reply:
let's remove it.